### DISCUSSION TITLE
Group exactly the same code

### DISCUSSION THREAD

**Discussion by ksyx** [community/community#10294] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/18738953/150142893-0d1c3268-edb7-4bf8-ae91-a58bb4315f47.png)
This looks not too helpful if I wanted to see something other than `parseParens()` 🤔

**Comment by colinwm** [comment]
Interesting idea... but there are a couple things you can do from that dialog:

1. Click "show more references" to see a longer list
2. Click "next" a bunch of times to go through the 27 matches
2. Look on the right gutter of the page, it'll show you the relative positions of the matches. If you click near ones on the bottom, this dialog will jump to show references lower down in the file

**Reply by ksyx** [reply]
Yes show more reference is a good thing but it is still not a complete list, with lots of duplications. I personally think removing duplication by grouping gives better experience and information, though this may break the design of ordering them in the order of file.
